src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrOfStrings = sourceString.split(';')
+    .map(item => item.split(':')
+      .map(prop => prop.trim()));
```

Comment:
don't use map inside another map, you can avoid it here

Reply:
Why i can't use it? :c

Reply:
you can, but that makes your code very slow, reed about algorithm complexity.